site/content/docs/tutorials/supervisor-without-concierge-demo.md

```diff
@@ -0,0 +1,265 @@
+# self-signed CA. Make this match where you mounted the CA PEM file into
+# your control plane node's filesystem, which must be under a directory
+# that the kube-apiserver container is going to volume mount.
+--oidc-ca-file="/etc/ca-certificates/supervisor/root-ca.pem"
+
+# Choose a unique value for each cluster here. By making this unique, the
+# Supervisor will be able to issue ID tokens for this cluster that cannot
+# be used on any other cluster, which improves security. Do not use the
+# special value "pinniped-cli" or any value that contains the substring
+# ".pinniped.dev", because these special values are reserved for other
+# purposes.
+--oidc-client-id="my-cluster-342klb7h"
+
+# Use these exact values. These are based on how the Supervisor issues ID
+# tokens. Do not change these values.
+--oidc-signing-algs="ES256"
+--oidc-username-claim="username"
+--oidc-groups-claim="groups"
+
+# These are optional, use any value you prefer here, or do not set these flags.
+# These strings will be prepended to the username and group name strings
+# that were determined by the Supervisor during user authentication to decide
+# the final username and group names, but only on this cluster. Refer to the
+# Kubernetes kube-apiserver docs for more information about these flags.
+--oidc-username-prefix="pinniped:"
+--oidc-groups-prefix="pinniped:"
+```
+
+Use the `--oidc-client-id` to choose a string that is unique for each cluster. This could be a GUID or some other random
+letters and numbers, and can be combined with a human-readable portion if desired. When a user first authenticates
+to the Pinniped Supervisor, it will issue an ID token with the `aud` (audience) claim set to the name of the client,
+which will be either `pinniped-cli` (for the kubectl use case) or will start with `client.oauth.pinniped.dev-`
+(for a web app client using the `OIDCClient` CR). Avoid using these names for the `--oidc-client-id` value to ensure
+that these initial ID tokens cannot be used to authenticate to your cluster. Next, the client will make another call
+to the Supervisor to obtain a new ID token which is scoped to one specific cluster. This new token will have the `aud`
+claim's value changed to the cluster's unique value. This is the only token that will be sent to that cluster.
+The `--oidc-client-id` flag of `kube-apiserver` tells it to validate the `aud` claim on the incoming ID tokens.
+This cluster-scoped ID token will not be accepted by any other cluster, because no other cluster should use the
+same unique value for this flag. This improves the security of your clusters by making this token only valuable on
+a single cluster.
+
+The procedure to add these command-line flags to the `kube-apiserver`'s list of command-line flags depends on
+the distribution of Kubernetes that you are using. Please refer to the documentation for your distribution.
+
+Note that you can configure these flags even if the Supervisor is not running yet. The Kube API server will
```

Comment:
"the Pinniped Supervisor" to be consistent

Reply:
Fixed.